src/router/validation/ValidationUtilities.ts

```diff
@@ -0,0 +1,9 @@
+import { Validator } from 'fhir-works-on-aws-interface';
+
+export async function validateResource(validators: Validator[], resource: any) {
+    return Promise.all(
+        validators.map((validator: Validator) => {
```

Comment:
can you use `validators.every` instead https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every

Reply:
I'm going to change this to use a for loop that runs the validator sequentially as per @carvantes comment about deterministic error.